test/unit/store/model-store.js

```diff
@@ -130,29 +155,29 @@ describe('ModelStore', () => {
             models = ModelStore.getModels();
             let array = models.filter(t => t.address === deleteModelAddress);
             assert.lengthOf(array, 0);
-            assert.strictEqual(models.length, 0);
+            assert.strictEqual(models.length, 1);
         });
 
     });
 
     it('should fail to delete a model', () => {
-        assert.strictEqual(models.length, 1);
+        assert.strictEqual(models.length, 2);
         return addAction('DELETE_MODEL', {
             modelAddress: 'This is not an address'
         }).then(assert.fail).catch(e => {
             assert.strictEqual(e.message, 'Unknown type.');
-            assert.strictEqual(models.length, 1);
+            assert.strictEqual(models.length, 2);
         });
     });
 
     it('should notice invalid modelAddress', () => {
-        assert.strictEqual(models.length, 1);
-        let address = 'm8y7nFLsT';
+        assert.strictEqual(models.length, 2);
+        let address = createAddress(100, 'model');
```

Comment:
if we every have 100 models in our test data for this file, this would start to fail again. Probably fine